### PR TITLE
Adding support of multiple operators in one fragment

### PR DESCRIPTION
**Pull request by lipoja**
Adding support of multiple operators in one FBC fragment.

[CLOUDDST-22242]

### CODE REVIEW

**Review by JAVGan — APPROVED** [review]

**Review by yashvardhannanavati — DISMISSED** [review]

File: iib/workers/tasks/fbc_utils.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def merge_catalogs_dirs(src_config: str, dest_config: str):
     shutil.copytree(src_config, dest_config, dirs_exist_ok=True)
 
 
-def extract_fbc_fragment(temp_dir: str, fbc_fragment: str) -> Tuple[str, str]:
+def extract_fbc_fragment(temp_dir: str, fbc_fragment: str) -> Tuple[str, List[str]]:
     """
     Extract operator package from the fbc_fragment image.
```
Comment (chandwanitulsi) [inline iib/workers/tasks/fbc_utils.py:96]:
```suggestion
    Extract operator packages from the fbc_fragment image.
```

File: iib/workers/tasks/fbc_utils.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def merge_catalogs_dirs(src_config: str, dest_config: str):
...
```
Comment (chandwanitulsi) [inline iib/workers/tasks/fbc_utils.py:97]: return: fbc_fragment path, fbc_operator_package
```suggestion
return: fbc_fragment path, fbc_operator_packages
:rtype: list
```

File: iib/workers/tasks/fbc_utils.py [hunk 3]
```diff
@@ -114,7 +114,5 @@ def extract_fbc_fragment(temp_dir: str, fbc_fragment: str) -> Tuple[str, str]:
     log.info("fbc_fragment contains package %s", operator_packages)
```
Comment (chandwanitulsi) [inline iib/workers/tasks/fbc_utils.py:114]:
```suggestion
    log.info("fbc_fragment contains packages %s", operator_packages)
```

**Review by chandwanitulsi — COMMENTED** [review]
Added a few suggestions, overall LGTM

**Comment by lipoja** [comment]
Rebased and added @chandwanitulsi suggestions.

**Review by yashvardhannanavati — APPROVED** [review]

**Review by chandwanitulsi — APPROVED** [review]

**Review by yashvardhannanavati — APPROVED** [review]

**Pull request merged**